### ISSUE TITLE
Can't launch GUI

### ISSUE THREAD

**Issue by IsabelleSajonia** [shlizee/OpenLabCluster#4]
Hi. I saw a similar issue was raised before but as it hasn't been active recently and my problem may be different, I'm starting a new thread. I am using windows and an anaconda prompt (python 3.7.16). I was able to install everything ok (note I had to use pip install openlabcluster[gui] instead of pip install 'openlabcluster[gui]' because I was getting the error "invalid requirement"). All of this process looked fine but pythonw.exe -m openlabcluster does not launch a GUI for me. I can download Linux and try that but I would prefer Windows if possible. Thanks!

**Comment by IsabelleSajonia** [comment]
Update: I was able to successfully install everything in Linux but I still get an error when trying to launch the GUI using the execution in the readme. 
![image](https://github.com/shlizee/OpenLabCluster/assets/70863857/82799848-b8c6-454b-a4c0-b8d1d50b5c3c)

I also tried pulling from github (though I'm not as familiar with this and may have made a mistake) and when trying to launch the GUI get this error: 
![image](https://github.com/shlizee/OpenLabCluster/assets/70863857/29d229aa-ce18-4121-9197-50f47922242a)

**Comment by christincha** [comment]
Hi, did you install with pip install? Could you go to the folder where you install openlabcluster /irs3th/OpenLabCluster/openlabcluster to see what's in there?

**Comment by IsabelleSajonia** [comment]
yes, I used pip install and here's a screenshot of the folder. Thanks for the quick reply! 

![image](https://github.com/shlizee/OpenLabCluster/assets/70863857/746d65cf-d4f6-43a2-8a95-3a4ff1f6ca88)

**Comment by christincha** [comment]
Is this the folder you downloaded from GitHub? Could you go to the environment path like /home/**/anaconda3/envs/OpenLabCluster/lib/python3.7/site-packages/openlabcluster to check if the openlabcluster is successfully installed?

**Comment by IsabelleSajonia** [comment]
I'm not sure about the git hub download I don't see that path but for the Linux download instructions in the readme when I go to the OpenLabCluster environment I can see that openlabcluster is listed in my packages 
![image](https://github.com/shlizee/OpenLabCluster/assets/70863857/4a55d222-7379-4803-b225-e2b3d97cef72)

**Comment by christincha** [comment]
Hi, did you follow the exact instructions here, including creating with spec-file.txt?: 

```
conda create --name OpenLabCluster --file spec-file.txt
conda activate OpenLabCluster
pip install openlabcluster
```

Which Linux system version are you using?
